youtube_dl/extractor/gamestar.py

```diff
@@ -38,11 +44,12 @@ def _real_extract(self, url):
             webpage, 'JSON-LD', group='json_ld'), video_id)
         info_dict = self._json_ld(json_ld, video_id)
         info_dict['title'] = remove_end(info_dict['title'], ' - GameStar')
+        info_dict['title'] = remove_end(info_dict['title'], ' - GamePro')
```

Comment:
46-47 code duplication.

Reply:
Any preferences on how to solve this? I can squeeze it in one line with two calls to `remove_end`, but in my eyes this make things only worse.

Reply:
The point is eliminating code duplication, not squeezing into one line.

Reply:
I don't really get what you want. I could remove the code duplication by defining a micro-function calling remove_end, or I could write my own remove_end which removing based on a regex or something like this. In my eyes neither change provides any benefit.
Please give me a direction on how I should reformat the code, I think we have different opinions on coding style.